langchain/document_loaders/telegram.py

```diff
@@ -217,10 +217,11 @@ def _combine_message_texts(
 
     def load(self) -> List[Document]:
         """Load documents."""
-        if self.chat_url is not None:
+        import pandas as pd
```

Comment:
we should keep the import within the try except

Reply:
well really we should probably add a separate try/except for when we first use pandas (looks like L239)

Reply:
I can add it a separate one though, just haven't seen such type of imports in a while.

Reply:
fixed